game_data.py

```diff
@@ -12,7 +16,21 @@
     }
 }
 
-players = {}
+players: List[Player] = []
 
 # start the server
 mud = MudServer()
+
+
+class Game():
+    def __init__(self):
+        self.players: Dict[str, Player] = {}
+
+    def add_player(self, player: Player):
+        self.players[player.uuid] = player
```

Comment:
Somewhat off topic, but I played about with a `def __str__` for the `Client`/`Player` class a bit, which returned the uuid, so we could say `player` as a shorthand for `player.uuid`. Not sure if it's a good pattern though, maybe being explicit is better? Dunno

Reply:
I think explicit is better in this case.